proto/defs.bzl

```diff
@@ -71,7 +71,7 @@ def ts_proto_library(name, proto, **kwargs):
         # https://github.com/protobufjs/protobuf.js/tree/6.8.8#pbjs-for-javascript
         args = [
             "--target=static-module",
-            "--wrap=default",
```

Comment:
This change is unrelated -- it just outputs more readable generated TS proto code.